hackathon_site/event/serializers.py

```diff
@@ -86,6 +86,22 @@ def update(self, instance: Profile, validated_data):
         return serializers.ModelSerializer.update(self, instance, validated_data)
 
 
+class CreateProfileSerializer(ProfileSerializer):
+    class Meta(ProfileSerializer.Meta):
+        read_only_fields = ("id", "team", "id_provided", "attended")
+
+    def create(self, validated_data):
+        response_data = {"attended": True}
+        response_data["user"] = self.context["request"].user
```

Comment:
you don't need this since your serializer doesn't include user

Reply:
Yep! I have resolved most of the issues brought up, when I try to remove user though, it complains that it is missing a mandatory user_id within the request.